popup.css

```diff
@@ -0,0 +1,185 @@
+.projectPopup {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  background-color: rgba(193, 199, 208, 0.9);
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project1 {
+  background-color: #fff;
+  height: 97%;
+  width: 92%;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224512302-dd2f75e9-31e0-4344-9ba4-f5fc25d1d160.png)

**[OPTIONAL]**

- [ ] Kudos on making the requested changes by the previous reviewer but I would apologize on behalf of the previous reviewer because he didn't tell you about your mobile popup design on 375px screen size 😨 As you can see in the image above that your design is not professional at all 🤔 Kindly fix it to make your project attractive for the viewers of your repository especially the recruiters 👍